test/motor.js

```diff
@@ -2082,27 +2089,34 @@ exports["Motor.Collection"] = {
   },
 
   callForwarding: function(test) {
-    test.expect(3);
+    test.expect(4);
 
     var motors = new Motor.Collection([{
-      pwm: 3,
-      dir: 4
+      pins: {
+        pwm: 3,
+        dir: 4
+      }
     }, {
-      pwm: 5,
-      dir: 6
+      pins: {
+        pwm: 5,
+        dir: 6
+      }
     }, {
-      pwm: 9,
-      dir: 10
+      pins: {
+        pwm: 9,
+        dir: 10
+      }
```

Comment:
I guess this should also check and make sure that the Motor objects are constructed correctly

Reply:
I added tests that make sure each member of the collection is an instance of Motor.